app/models/season.rb

```diff
@@ -46,8 +46,20 @@ def ended?
 
   def year; name end
 
-  # switch_phase, in dev env only: enables developers to easily switch
+  # switch_phase: enables developers to easily switch
   # between time dependent settings in views
+  def fake_proposals_phase
+    update({
```

Comment:
You can use Rails' `from_now` so that e.g. `Date.today+6.months` becomes `6.months.from_now`